crates/cxx-qt-lib-headers/include/common.h

```diff
@@ -51,5 +51,12 @@ operatorEq(const T& a, const T& b)
   return a == b;
 }
 
+template<typename T>
+int8_t
```

Comment:
elsewhere we have been using fully qualified types to avoid any namespace fun later, so `::std::int8_t` ?